server/engagement/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from members.models import User
+
+class AttendanceSession(models.Model):
+    id = models.AutoField(primary_key=True)
```

Comment:
changed my mind we probably should prefix this since we're doing it elsewhere, e.g. session_id